thesis-de/content-de.tex

```diff
@@ -1094,56 +1094,221 @@ \subsubsection{Durchführung Evaluierung Frontend}
 Im Nachfolgenden Punkt sind die einzelnen Softwarelösungen mit den einzelnen
 Anforderungen und Evaluierungsergebnissen aufgelistet:
 
-
+\paragraph{Grafana}
+\label{paragraph:grafana}
+Grafana ist eine von Torkel Ödegaard & Raintank Inc Entwickelte OpenSource
+Visualisierungssoftware. Sie bietet neben einfachen Visualisierungs Graphen noch
+weitere Ereignisfunktionen. Es arbeitet mit jedem TimeSeries
+Datenhaltungssystem, wie z.B. influxDB zusammen und ist Multi Tenant fähig.
+Dies bedeutet, dass sowohl ein LDAP als auch eine Benutzernamen plus Kennwort
```

Comment:
Habe ich angepasst. LDAP kommt noch ins Glossar. anschließend Referenziere ich auf alle LDAP kürzel in ther Thesis